CHANGELOG.md

```
@@ -0,0 +1,505 @@
+- Search results not respecting user-specified `limit` parameter (now defaults to 5) [#98]
+
+### Added
+
+#### Qdrant-loader-mcp-server
+
+- Optional `similarity_threshold` parameter (default 0.7) to `find_similar_documents` for filtering by minimum similarity score [#91]
+- Separate `fetch_limit` from user `limit` with configurable defaults for better control [#98]
+
+### Changed
+
+#### Qdrant-loader-mcp-server
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)